D1673/P1673.bs

```diff
@@ -5008,22 +5273,15 @@ auto operator* (const T1 x, const conjugated_scalar<Reference, Element> y) {
 
 * *Preconditions:* `Reference` shall be *Cpp17CopyConstructible*.
 
-* *Constraints:* The expression `conj(val)` is well formed and is convertible to
-    `ElementType`.  *[Note:* This implies that `ElementType` is
-    `complex<R>` for some type `R`.  --*end note]*
+* *Constraints:* The expression _`conj-if-needed`_`(val)` is well formed
+    and is convertible to `ElementType`.
```

Comment:
I think there's a spelling mismatch here of `Element` vs `ElementType`

Reply:
Good catch!  I'll fix this in #242.  Thanks!